### ISSUE TITLE
[Other - Accessibility] Keyboard focus doesn't move as appropriate to workflow. (00.00.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#73992]
#### VFS team name
Benefits Non-Disability

#### VFS product name
Pension 527EZ Digital Form

#### VFS feature name
Form Reactivation

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On [Gross monthly income](https://staging.va.gov/pension/application/527EZ/financial/income-sources), when entering multiple income sources in the list-and-loop, after pressing the Add another income source button keyboard focus doesn't move to the top of the new income source group of inputs.

For a screen reader user, nothing is announced at all --- which may make you question if it worked, try to press enter again, etc. The next tab stop is the Back button (missing the new fields), and a user may erroneously go back, then move forward again.

This was logged as an issue by @vbellisimo in #73299, and it looks like @ToddWebDev may already have a solution in the works. Hopefully this one is an easy one!

This also occurs on:
- [Add an unreimbursed care expense](https://staging.va.gov/pension/application/527EZ/financial/care-expenses/add)

### Link, screenshot or steps to recreate
Steps to reproduce:
1. Activate a screen reader (NVDA on Windows, VoiceOver on Mac).
2. Navigate through the form to [Gross monthly income](https://staging.va.gov/pension/application/527EZ/financial/income-sources).
3. Enter your first income source.
4. Press tab to move focus to Add another income source button.
5. Press enter.
6. Note that nothing is announced by your screen reader.
7. Press tab.
8. Note that keyboard focus moves to the Back button.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Other list-and-loops in this workflow don't have this issue, so I'm not sure what might be happening here. Please feel free to reach out to me if you need any help testing or validating your solution.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by aplatt-coforma** [comment]
Moving this to Review/QA since the fix for the related (duplicate) ticket #73299 is already on Staging.

**Comment by gmaciolek** [comment]
This appears to be resolved by @ToddWebDev 's fix. When I checked it out on staging, after pressing the "Add another income source" button, the keyboard focus goes to the first radio button in the “What type of income?” question in the New income source section. That’s also what’s being announced in VoiceOver on Mac.

**Comment by ToddWebDev** [comment]
This ticket is resolved with the work completed for internal Issue #73299: https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/gh/department-of-veterans-affairs/va.gov-team/73299

**Comment by shiragoodman** [comment]
@aplatt-coforma Are you asking for Governance to help validate that this issue is resolved?

**Comment by aplatt-coforma** [comment]
@shiragoodman - No, I don't think that is needed to close the ticket right? Per the VFS Guidelines, we just need to validate the fix or get our Product Owner to review. 

Since both @gmaciolek and @ToddWebDev have verified it, I think we can close today. Just moved it to review column for our team's visibility.

**Comment by shiragoodman** [comment]
yup @aplatt-coforma ! that's correct. close when you're ready :)
